src/test/java/pages/BasePage.java

```diff
@@ -14,6 +20,14 @@ public BasePage(WebDriver driver, String url) {
     public void open() {
         driver.get(url);
     }
+    public void blurElement(WebElement element) {
+        ((JavascriptExecutor)driver).executeScript("arguments[0].blur()", element);
+    }
+
+    public void waitForElementToBeClickableAndClick(WebElement element) {
+        new WebDriverWait(driver, Duration.ofSeconds(10))
```

Comment:
We can get timeout value that we selected during test setup phase.
```suggestion
        new WebDriverWait(driver, driver.manage().timeouts().getImplicitWaitTimeout()).
```

Remove unused import in case of replacement.